app/views/home/index.html.slim

```diff
@@ -1,32 +1,91 @@
-section#speakers
+section#schedule
   .container
     h2.section-header
-      ' Invited Speakers
+      ' Schedule
```

Comment:
This likely has no practical impact (except for maybe alignment in case of centred text) but `'` adds a trailing whitespace, whereas `|` doesn't. 🙂